kolibri/utils/options.py

```diff
@@ -62,7 +63,7 @@ def calculate_thread_pool():
     if psutil:
         MIN_MEM = 2
         MAX_MEM = 6
-        total_memory = psutil.virtual_memory().total / pow(2, 30)  # in Gb
+        total_memory = psutil.virtual_memory().total / pow(10, 9)  # in Gb
```

Comment:
for full pedantic correctness :D

```suggestion
        total_memory = psutil.virtual_memory().total / pow(10, 9)  # in GB
```